crit-go/gocrit/cmdhandler.go

```diff
@@ -0,0 +1,120 @@
+package gocrit
+
+import (
+	"crit-go/images"
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+func Check(e error) {
+	if e != nil {
+		fmt.Fprintln(os.Stderr, e)
+		os.Exit(1)
+	}
+}
+
+func checkfile(e error, f *os.File) {
+	if e != nil {
+		f.Close()
+		fmt.Fprintln(os.Stderr, e)
```

Comment:
This error message will be non-informative. It will not say what operations triggers an error. I don't like these checkfile and Check functions, this isn't go-style.

Reply:
okay, i will remove this.

Reply:
i have removed these functions replacing them with logging